testsets/pc_windows.yaml

```diff
@@ -197,3 +197,10 @@
   os: Windows XP
   os_version: 'NT 5.1'
   category: pc
+
+#YaBrowser + Windows
+- target: 'Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 YaBrowser/18.1.1.839 Yowser/2.5 Safari/537.36'
+  name: Yandex Browser
+  os: Windows 7
+  os_version: 'NT 6.1'
+  category: pc
```

Comment:
I think that need `version`, because `name` is present and `version` is clear.
(In this case `version` is `18.1.1.839`.)

P.S.
I wrote this feature in Rust, and this test case is fail.
https://travis-ci.org/woothee/woothee-rust/jobs/366522741#L675

Reply:
add https://github.com/woothee/woothee/pull/44